src/components/ui/cardComponent.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "../../assets/cardComponent.css";
+export class Card extends React.Component{
+    render(){
+        var card=(<></>);
```

Comment:
IMO this is not the best thing to do, better to make an early return for when you have a cart type "category" and in all the other instances just return the normal card

```
export const Card = ( {cardType} ) => {
   if (cardType === 'category') return category card // this includes your div with the class card
   return (
      <div className="card">
                // normal card here 
       </div>    
   )
}
```